### PR TITLE
Fix for #1958

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

**Comment by Horneth** [comment]
👍

**Comment by mcovarr** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ before_cache:
   - find $HOME/.sbt -name "*.lock" -delete
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=rm_missingImportsTest
```
Comment (cjllanwarne) [inline .travis.yml]: Remember to revert before merging!

Comment (ruchim) [inline .travis.yml]: I needed that reminder...

**Comment by geoffjentry** [comment]
👍

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala [hunk 2]
```diff
@@ -185,6 +185,7 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
         |cd $cwd
         |${globManipulations(globFiles)}
         |)
+        |sync
```
Comment (Horneth) [inline backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: It's curious that this is here, I already merged this PR in develop

**Comment by cjllanwarne** [comment]
This seems to have brought along a few other PRs somehow? Like @Horneth and the `wdlDependencies` thing

File: project/Dependencies.scala [hunk 3]
```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   lazy val lenthallV = "0.21-e1b7822-SNAP"
-  lazy val wdl4sV = "0.10-cc58e9b-SNAP"
+  lazy val wdl4sV = "0.10-dce653c-SNAP"
```
Comment (cjllanwarne) [inline project/Dependencies.scala]: If you do merge this, when you rebase, please keep this at the latest wdl4s to include my changes- `9ad47e0`

**Comment by ruchim** [comment]
The fix for #1958 can be found in https://github.com/broadinstitute/cromwell/pull/1975. Closing this PR which is no longer needed.

**Pull request closed without merging**